### ISSUE TITLE
DOCS: not working

### ISSUE THREAD

**Issue by sreichel** [OpenMage/magento-lts#4439]
Please test https://docs.openmage.org/.

**Comment by sreichel** [comment]
https://www.openmage.org/magento-lts/ works.

Maybe leave it as it is and rewrite https://www.openmage.org/magento-lts/ to https://www.openmage.org/docs/

**Comment by theroch** [comment]
Works for me https://docs.openmage.org/

**Comment by sreichel** [comment]
@colinmollenhour? Did you updated/changed anything?

**Comment by colinmollenhour** [comment]
Yes, it was both caused and reported by a HackerOne user - I resolved it by validating the openmage.org domain so that other accounts cannot "steal" the custom domain so it shouldn't happen again.

**Comment by sreichel** [comment]
Perfect, Thanks.

**Comment by sreichel** [comment]
@colinmollenhour Its broken again.

**Comment by theroch** [comment]
I can confirm it. Site response is 404 "Site not found".

**Comment by colinmollenhour** [comment]
I don't know how this is happening, but it appears to be related to https://github.com/OpenMage/magento-lts/actions/runs/13189044903.

Before:

![Image](https://github.com/user-attachments/assets/23d9203f-e32e-4fbc-86d6-6465cd966cf1)

When I changed the CNAME it created this commit (I didn't do this manually):

![Image](https://github.com/user-attachments/assets/0c1bed7f-5df7-4a03-a221-1caf011815b4)

However, I can't find anything that would explain how it is reverted back to www.openmage.org.. Perhaps this `CNAME` file should be created by the `mkdocs gh-deploy --force` in the `docs.yml` workflow?

**Comment by colinmollenhour** [comment]
Ahh, according to [this](https://www.mkdocs.org/user-guide/deploying-your-docs/#custom-domains) there should be a CNAME file in or docs_dir which is docs/content. I just added this file in c7c3d99b35b0b.

**Comment by sreichel** [comment]
Thanks again.

**Comment by sreichel** [comment]
CNAME file in root is obsolete?

**Comment by colinmollenhour** [comment]
> CNAME file in root is obsolete?

I don't think so?  Why do you say that?

**Comment by sreichel** [comment]
Thought its used from https://github.com/OpenMage/magento-lts/tree/main/docs/content.

**Comment by colinmollenhour** [comment]
It is supposed to be in the docs_dir so it is copied to the root in the generated output.
